docs/man/docker-cp.1.md

```diff
@@ -1,17 +1,50 @@
 % DOCKER(1) Docker User Manuals
 % Docker Community
-% JUNE 2014
+% JANUARY 2015
 # NAME
-docker-cp - Copy files/folders from the PATH to the HOSTPATH
+docker-cp - Copy files/folders between containers or a local path
 
 # SYNOPSIS
 **docker cp**
-CONTAINER:PATH HOSTPATH
+[SRC_CONTAINER:]SRC_PATH [DST_CONTAINER:]DST_PATH
 
 # DESCRIPTION
-Copy files/folders from a container's filesystem to the host
-path. Paths are relative to the root of the filesystem. Files
-can be copied from a running or stopped container.
+Copy files/folders to or from a container and your local host
+or between any two containers. Container paths are relative to
+the container's working directory. Files can be copied to and
```

Comment:
A container's working directory can also be set with the `-w, --workingdir=""` option to `docker run` or `docker create`. A client can query for the working directory of a container using `docker inspect CONTAINER` and view the `Config.WorkingDir` entry.

Here's an example with the new `docker cp` and a relative path:

```
$ cID=$(docker run -d -w /root ubuntu /bin/sh -c "echo Hello, World\! > hello.txt")
$ docker inspect $cID | grep WorkingDir
        "WorkingDir": "/root"
$ docker cp $cID:hello.txt ./hello_container.txt
$ cat ./hello_container.txt
Hello, World!
$ docker cp $cID:/root/hello.txt ./hello_container_from_root.txt
$ cat ./hello_container_from_root.txt 
Hello, World!
```